apps/src/gamelab/GameLab.js

```diff
@@ -48,9 +48,16 @@ import {
 } from '../containedLevels';
 import { hasValidContainedLevelResult } from '../code-studio/levels/codeStudioLevels';
 import {actions as jsDebugger} from '../lib/tools/jsdebugger/redux';
+import project from '@cdo/apps/code-studio/initApp/project';
 
 var MAX_INTERPRETER_STEPS_PER_TICK = 500000;
 
+// The width and height in pixels of the thumbnail image to capture.
+const THUMBNAIL_SIZE = 180;
+
+// Number of ticks after which to capture a thumbnail image of the play space.
+const CAPTURE_TICK_COUNT = 250;
```

Comment:
Is 250 ticks really about one second after the run button is clicked?  Even at 60fps this ought to be more than 4 seconds into execution, unless I'm misunderstanding this.